### PR TITLE
Add reevoo-click-action and on-click badge configuration to documentation

### PR DESCRIPTION
**Pull request by robinbortlik**
- ticket: https://reevoo.atlassian.net/browse/EX-2321

### CODE REVIEW

File: docs/reevoomark/product-badges.markdown [hunk 1]
```diff
@@ -102,3 +102,30 @@ We strongly recommend using more specific parameters such as `model-year`, `body
...
+
+Available options are `open_lightbox`, `open_window`, `no_action`.
+
+{% highlight html %}
+<reevoo-reviewable-badge trkref="TRKREF" sku="SKU" reevoo-click-action="open_lightbox"></reevoo-reviewable-badge>
+{% endhighlight %}
+
+## Custom click callback
+
+In cases when you need to implement your on logic around click action, we give you ability to define your custom javascript function. This function will be evaluated when user click on the badge, but **after** standard click action is executed. That means after lightbox is opened. You can use `on-click` callback together with `reevoo-click-action` setting.
```
Comment (costis) [inline docs/reevoomark/product-badges.markdown]: your "own" logic

Comment (costis) [reply docs/reevoomark/product-badges.markdown]: we give you "the" ability

Comment (costis) [reply docs/reevoomark/product-badges.markdown]: when user "clicks" on the badge

Comment (robinbortlik) [reply docs/reevoomark/product-badges.markdown]: Ahh... nice catch @costis thank you very much.

**Pull request merged**